### ISSUE TITLE
Problem to install jasmin-sms-gateway 0.10.11 on ubuntu 20.04

### ISSUE THREAD

**Issue by FlorentinCONGO** [jookies/jasmin#1033]
Hi, i tried to install Jasmin sms gateway but i can't. When i use the commands on https://docs.jasminsms.com/en/latest/installation/index.html#installation-prerequisites, i got: cannot find package jasmin-sms-gateway.
If i use this commands: `curl -s https://packagecloud.io/install/repositories/jookies/jasmin-sms-gateway/script.deb.sh | sudo bash` i got the message:
 ```
Paramétrage de python3-txredisapi (1.4.8) ...
ERROR: Could not find a version that satisfies the requirement txredisapi==1.4.8 (from versions: 0.2, 0.3, 0.5, 0.6, 0.8, 0.9, 1.0, 1.1, 1.2, 1.3, 1.4, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.7)
ERROR: No matching distribution found for txredisapi==1.4.8
```
Could you help me please! 
Thanks in advance.

**Comment by AnythingLinux** [comment]
Try to use Ubuntu 20.04 LTS 64 bit Linux.
Install python 3.8.x version and set default python version.
Install pip pip 20.0.2 version and set default python version.
Install Python: https://gist.github.com/UbuntuEvangelist/fe805176e61d927d11b139166bdbfd6d
After install python 3.8.x version follow:

apt install python3-pip
cd
nano ~/.bashrc
# Add below to lines into the bottom of ~/.bashrc file
alias python=python3
alias pip=pip3
# Save & Exit ctrl+s and ctrl+x
alias python=python3
alias pip=pip3
source ~/.bashrc
sudo apt update
sudo apt upgrade -y
python --version
pip --version
pip install txredisapi
apt-get install python3-venv
apt install redis rabbitmq-server -y
pip install gunicorn
apt install pipx 
pipx install twisted --force
pipx ensurepath

curl -s https://packagecloud.io/install/repositories/jookies/jasmin-sms-gateway/script.deb.sh | sudo bash
apt-get install jasmin-sms-gateway
apt update -y && apt upgrade -y 

OR

curl -s https://setup.jasminsms.com/deb | sudo bash
apt install jasmin-sms-gateway
apt update -y && apt upgrade -y 

systemctl enable jasmind
systemctl start jasmind
systemctl status jasmind


# Contact telegram @LinuxGun for bulk sms gateway development using Jasmin SMS Gateway

**Comment by FlorentinCONGO** [comment]
> 

hi @UbuntuEvangelist, thanks for your help. I tried to install jasmin sms with your instructions but when i try to install it i got: 
![Sans titre 1](https://user-images.githubusercontent.com/30045077/156586469-e8424009-77e1-45bd-beac-9674b56f211b.jpg)

this command `pip install txredisapi` install version **1.4.7** or not **1.4.8** .  So whent installation failed i have the message 
![Sans titre 2](https://user-images.githubusercontent.com/30045077/156587226-a047b860-1c88-42b4-9968-0c170d73c7c7.jpg)

thanks for your help!

**Comment by AnythingLinux** [comment]
@FlorentinCONGO 
from project folder run below: 
pip install txredisapi==1.4
pip install python3-txredisapi
pip install jasmin
or
pip install jasmin-sms-gateway
pip install requirements.txt

**Comment by farirat** [comment]
@FlorentinCONGO what os and version ?

**Comment by farirat** [comment]
Issue fixed and tested on ubuntu 20.04, 21.10 and RHEL 8
